packages/app-graphql/README.md

```diff
@@ -5,4 +5,60 @@ title: GraphQL API
 draft: true
 [meta]-->
 
-# GraphQL API
+# GraphQL App
+
+A Keystone App that creates a GraphQL API and Apollo GraphQL playground.
+
+For information about writing queries and mutations for Keystone see the [Introduction to Keystone's GraphQL API](/guides/intro-to-keystone).
```

Comment:
Is this meant to link to the doc above?

Also, should this be an absolute URL like all the others?

```suggestion
For information about writing queries and mutations for Keystone see the [Introduction to Keystone's GraphQL API](https://v5.keystonejs.com/guides/intro-to-graphql).
```